javascript/javascript1/week4/sentimentAnalizer.js

```diff
@@ -0,0 +1,48 @@
+
+const positiveList = [
+    "happy",
+    "super",
+    "awesome",
+    "great",
+    "delightful",
+    "cool",
+    "fabulous",
+    "beautiful",
+    "brilliant",
+    "fun"
+  ];
+  const negativeList = [
+    "angry",
+    "bad",
+    "evil",
+    "awful",
+    "ugly",
+    "upset",
+    "sad",
+    "boring",
+    "hate",
+    "hard"
+  ];
+  
+  function getSentimentScore(feelingsArr) {
+    const feelingsConverted = feelingsArr.toLowerCase();
+    const wordByword = feelingsConverted.split(" ");
+    const positiveResult = [];
```

Comment:
Not sure if there is a point of having arrays, as you're interested only in the number of words. You could instead use 2 numbers and increment them according to your if/else blocks.